remoulade/broker.py

```diff
@@ -104,6 +108,28 @@ def change_broker(broker: "Broker") -> None:
     declare_actors(actors)
 
 
+def set_tracing_middleware(middleware: "Middleware") -> None:
```

Comment:
I guess I would prefer if it's called add_middleware, this way it's not linked to the tracing, also it would call broker.add_middleware if broker exists.